src/core/utils/debug-utils.ts

```
@@ -0,0 +1,23 @@
+            b.toString(16).padStart(2, "0") + ((j + 1) % 8 === 0 ? "  " : " ")
+        )
+        .join("")
+        .padEnd(bytesPerLine * 3 + Math.floor(bytesPerLine / 8), " ");
+      const ascii = chunk
+        .map((b) => (b >= 0x20 && b <= 0x7e ? String.fromCharCode(b) : "."))
+        .join("");
+      lines.push(`${i.toString(16).padStart(4, "0")}: ${hex} ${ascii}`);
+    }
+    return lines.join("\n");
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)